learn/user-guide/why-ballerina/network-aware-type-system.md

```diff
@@ -20,9 +20,9 @@ With more services being available in the cloud, the network-distributed program
 
 ## Statically Typed and Structural
 
-Ballerina is a statically typed language, which means all the variables are checked at compile-time and only compatible values are assigned. Statically typed languages are generally more robust, easier to debug, and aids in creating better language tooling.
+Ballerina is a statically typed language, which means type compatibility between constructs are checked at compile-time. Statically typed languages are generally more robust to changes and refactorings, easier to debug, and aids in creating better language tooling.
 
-Ballerina’s type system is also structural as opposed to nominal. This means that the type compatibility is identified by considering the structure of the value. This is different from languages like Java, C++, and C# that have nominal type systems where it is bound by the name of the actual type.
+ The type system of Ballerina language is primarily structural with added support for nominal typing. This means that the type compatibility is identified by considering the structure of the value, rather than just relying on the name of the type. This is different from languages like Java, C++, and C# that have nominal type systems where it is bound by the name of the actual type.
```

Comment:
```suggestion
 The type system of the Ballerina language is primarily structural with added support for nominal typing. This means that the type compatibility is identified by considering the structure of the value rather than just relying on the name of the type. This is different from languages like Java, C++, and C# that have nominal type systems in which it is bound by the name of the actual type.
```